consent-ws/src/main/java/org/broadinstitute/consent/http/service/DataAccessParser.java

```diff
@@ -197,6 +199,20 @@ public PDAcroForm fillDARForm(Document dar, Map<String, String> researcherProper
         return acroForm;
     }
 
+    private String parseDatasetDetail(Document dar) {
+        ArrayList<Document> datasetDetail = (ArrayList<Document>) dar.get(DarConstants.DATASET_DETAIL);
+        Map<String, String> datasetDetailMap = new HashMap<>();
+
+        datasetDetail.forEach((doc) -> {
+            String objectId = doc.getString(DarConstants.OBJECT_ID) != null ? " | ".concat(doc.getString(DarConstants.OBJECT_ID)) : "";
+            datasetDetailMap.put(doc.getString("name"), objectId);
+        });
+
+        return datasetDetailMap.entrySet().stream()
+                .map(name -> name.getKey() + name.getValue())
+                .collect(Collectors.joining("; "));
```

Comment:
previously joined ids with ", " now joining with "; "
was this done on purpose?